packages/common/test/content/fetch.test.ts

```diff
@@ -817,33 +803,58 @@ describe("fetchHubContent", () => {
     expect(chk.schedule).toBeDefined();
     expect(getScheduleSpy.calls.count()).toBe(1);
 
-    expect(chk.id).toBe(HOSTED_FEATURE_SERVICE_GUID);
-    expect(chk.owner).toBe(HOSTED_FEATURE_SERVICE_ITEM.owner);
-    expect(chk.serverExtractCapability).toBeTruthy();
+    expect(chk.id).toBe(PDF_GUID);
+    expect(chk.owner).toBe(PDF_ITEM.owner);
 
     expect(getItemSpy.calls.count()).toBe(1);
-    expect(getItemSpy.calls.argsFor(0)[0]).toBe(HOSTED_FEATURE_SERVICE_GUID);
-    expect(getServiceSpy.calls.count()).toBe(1);
-    expect(getServiceSpy.calls.argsFor(0)[0].url).toBe(
-      HOSTED_FEATURE_SERVICE_URL
-    );
+    expect(getItemSpy.calls.argsFor(0)[0]).toBe(PDF_GUID);
     // NOTE: the first call to fetchItemEnrichments is done by fetchContent under the hood,
     // while the second call is done by fetchHubContent. We only care about the second call here
     expect(fetchItemEnrichmentsSpy.calls.count()).toBe(2);
-    expect(fetchItemEnrichmentsSpy.calls.argsFor(1)[0]).toBe(
-      HOSTED_FEATURE_SERVICE_ITEM
+    expect(fetchItemEnrichmentsSpy.calls.argsFor(1)[0]).toBe(PDF_ITEM);
+    expect(fetchItemEnrichmentsSpy.calls.argsFor(1)[1]).toEqual(["metadata"]);
+  });
+
+  it("handles when an error is thrown", async () => {
+    spyOn(console, "warn").and.callFake(() => null);
```

Comment:
From what I can tell, the CI failure is stemming from here
```suggestion
    spyOn(console, "warn").and.callFake((): null => null);
```